### ISSUE TITLE
Video Auto Splitting

### ISSUE THREAD

**Issue by CryZe** [LiveSplit/LiveSplitOne#335]
While "normal" auto splitting won't work in the web version, what can actually work is video based auto splitting. The web has a fairly easy to use screen capturing API, which we could build this on.

**Comment by wooferzfg** [comment]
The image comparison portion of this sounds like something that livesplit-core should handle.

**Comment by CryZe** [comment]
Yeah absolutely. For the sploosh kaboom capturing opencv.js was used and it was really slow, so we absolutely want to do this in WASM or the GPU.
